sdks/python/tox.ini

```diff
@@ -189,7 +189,7 @@ commands =
   time {toxinidir}/scripts/run_pylint.sh
 
 [testenv:docs]
-extras = test,gcp,docs
+extras = test,gcp,docs,interactive
```

Comment:
Does docs need it?

Reply:
Yes, `docs` needs it because it's generating documentation for all source code. Kind of like a static scan. That's why `test` and `gcp` packages have also been included here.